keyboards/smolpad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,481 @@
+// Copyright 2025 Wolf Yuan <me at wolf-yuan.dev>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "bitmaps.c"
+
+#define LED_START 0
+#define LED_END   11
+#define TRANSITION_MS 500
+#define OLED_FRAME_RATE_MS 33  // ~30 FPS
+
+enum custom_keycode {
+    MODE_SW = SAFE_RANGE,
+    WORKSPACE_RIGHT,
+    WORKSPACE_LEFT,
+    SHORTCUT_COPY,
+    SHORTCUT_SELECT,
+    SHORTCUT_PASTE
+};
+enum encoder_mode {
+    MODE_GAME,
+    MODE_CHAT,
+    MODE_MEDIA,
+    MODE_AUX,
+    MODE_MAX
+};
+enum action_mode {
+    VOL_UP,
+    VOL_DOWN,
+    PREV,
+    NEXT,
+    PLAYPAUSE,
+    NEXT_SPACE,
+    PREV_SPACE,
+    COPIED,
+    SELECTED,
+    PASTED,
+    NONE
+};
+enum menu_state {
+    MENU_NONE,
+    MENU_ACTIVE
+};
+enum setting_option {
+    SETTING_STEELSERIES,
+    SETTING_NATIVE,
+    SETTING_MAX
+};
+
```

Comment:
As per the PR Checklist, `default` keymaps should be left pristine and as an easy starting place for users making their own keymap. You can either remove the custom functionality here, or move it to a `vendor` keymap.
```suggestion
```